markdown/src/main/java/it/niedermann/android/markdown/MarkdownUtil.java

```diff
@@ -437,25 +437,20 @@ public static String getMarkdownLink(@NonNull String text, @NonNull String url)
      * @param s Markdown string
      * @return Plain text string
      */
+    @RequiresApi(api = Build.VERSION_CODES.N)
     @NonNull
+    // CS304 issue link: https://github.com/stefan-niedermann/nextcloud-notes/issues/1212
```

Comment:
Please avoid this kind of links, it will be visible in the commit messages if you tag the issue like
> #1212 Use Renderer to remove markdown